src/StimulusBundle/src/Dto/StimulusAttributes.php

```diff
@@ -212,18 +212,18 @@ private function getFormattedValue(mixed $value): string
         return (string) $value;
     }
 
-    private function escapeAsHtmlAttr(mixed $value): string
+    private function escape(mixed $value, string $strategy): string
```

Comment:
Is there any place we don't set 'html' ?

Reply:
if you fix https://github.com/symfony/ux/pull/2180#discussion_r1765076980, there will be

Reply:
Yes!